scripts/version-management.js

```
@@ -0,0 +1,341 @@
+}
+
+function generateNextVersion(currentVersion, releaseType = 'patch') {
+  const [major, minor, patch] = currentVersion.split('.').map(Number);
+  
+  switch (releaseType) {
+    case 'major':
+      return `${major + 1}.0.0`;
+    case 'minor':
+      return `${major}.${minor + 1}.0`;
+    case 'patch':
+    default:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)